mne_bids/meg_bids.py

```diff
@@ -294,14 +325,12 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
                          'got %s' % type(raw_file))
     data_path = make_bids_folders(subject=subject_id, session=session_id,
                                   kind=kind, root=output_path,
-                                  overwrite=overwrite,
```

Comment:
why are we dropping the overwrite params for the folders?

Reply:
It will never get called from the `raw_to_bids` function.
I mentioned something about it a bit further up in the discussion here, but adding in the functionality for a user to specify that they want this value to be True wasn't going to be worth it/seemed unnecessary.
The `make_bids_folder` function can still have the option to overwrite the data, just not from the `raw_to_bids` function.

Reply:
@teonbrooks I think semantically `overwrite` param makes sense only for the files, not for folders. Folders should be automatically created if they do not exist. The only thing is, you may want to clear/remove the folder but you can equally do that using an `rm -rf` -- so probably not worth it. Wdyt?

@monkeyman192 you should update the docstring to reflect the fact that `overwrite` param works only for files.

Reply:
yeah, I would think `overwrite` of a folder would be `rm -rf`. I agree that there is no utility in overwriting an empty folder. if one were trying to redo a bids creation, they might need to clear the directory but i think for now, it's OK to remove the param.